go.mod

```diff
@@ -120,3 +120,6 @@ require (
 )
 
 replace github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.9.9-0.20210509160536-b752439803c1
+
+// https://github.com/maistra/api/pull/14
+replace maistra.io/api => github.com/jwendell/maistra-api v0.0.0-20210607202744-08d78193a717
```

Comment:
what's the plan with this replace?

Reply:
(i.e. do you plan to remove once https://github.com/maistra/api/pull/14#pullrequestreview-679713002 is merged and before this merges?)

Reply:
Code looks good to me. Will approve now pending the update for the two replaces.